monai/metrics/metric.py

```diff
@@ -79,9 +77,14 @@ def _compute_list(self, y_pred: TensorOrList, y: Optional[TensorOrList] = None):
         Execute the metric computation for `y_pred` and `y` in a list of "channel-first" tensors.
 
         The return value is a "batch-first" tensor, or a list of "batch-first" tensors.
-        When it's a list of tensors, each item in the list can represent (batched) class-wise metrics.
+        When it's a list of tensors, each item in the list can represent a specific type of metric values.
 
-        Note: subclass may enhance the operation with multi-threads to accelerate.
+        For example, `self._compute_tensor` may be implemented as returning a list of `batch_size` items,
```

Comment:
Hi Nic, this is to show the number of items in the list is `batch_size`, and in the later docstring each item has the length of `batch_size`.

Reply:
OK, I see.
Maybe you can try to make it more clear to avoid confusion, some thing like:
`returning a list of `batch-first` items, length of each item is same as `batch_size`?

Thanks in advance.

Reply:
I think it's clear that I use "`" to denote the size, and the double-quote for the word"batch_first"

Reply:
OK, I missed line 79.

Thanks.